src/ngraph/frontend/onnx_import/utils/broadcasting.cpp

```diff
@@ -185,14 +159,20 @@ namespace ngraph
         {
             const auto& left_shape = left->get_shape();
             const auto& right_shape = right->get_shape();
-            // Broadcast only _stack of matrices_ axes.
-            const auto& numpy_shapes = get_numpy_broadcast_shape(
-                Shape{std::begin(left_shape), std::next(std::end(left_shape), -2)},
+            // Make temporary Parameters to pass shapes into broadcast function.
```

Comment:
Can we refactor `get_numpy_broadcast_shapes` to split it not 2 functions: one which takes `NodeVector` another which takes `vector<Shape>`? That way you could pass `Shape`s here instead of making temporary `Parameter` nodes.